runtime/syntax/buzz.yaml

```diff
@@ -0,0 +1,53 @@
+filetype: buzz
+
+detect:
+    filename: "\\.buzz$"
+
+rules:
+    # keywords
+    - keywords: "\\b(range|namespace|typeof|type|zdef|void|static|var|const|extern|import|export|from|test|as|in|while|if|else|fun|try|catch|throw|constructor|return|switch|default|break|for|foreach|do|until|continue|resolve|resume|yield|any|out)\\b"
+      # special method names
+    - symbol.operator: "\\b(!|and|or|band|bor|xor|\\?\\?)\\b"
+      # definitions
+    - identifier: "def [a-zA-Z_0-9]+"
+      # keywords (declaration)
+    - statements: "\\b(enum|object|protocol)\\b"
+      # constants (true, false, null, this)
+    - constants: "\\b(true|false|null|this)\\b"
+      # constants (types)
+    - type: "\\b(ud|str|int|float|obj|fib|bool|pat|type|any)\\b\\??"
+      # decorators
+    - preproc: "^\\s*@[^(]*"
+      # operators
+    - symbol.operator: "(=|\\+=|\\-=|\\/=|\\*=)"
```

Comment:
Do `+=`, `-=`, `/=`, and `*=` operators actually exist in Buzz? I don't see them used in the examples and they are not in the Textmate grammar.